tests/lib/collectors/aws_volume_snapshot.py

```diff
@@ -2,17 +2,19 @@
 # -*- coding: utf-8 -*-
 """Tests the GoogleCloudCollector."""
```

Comment:
not something quite within scope but it stands out now :)

 Should this be AwsVolumeSnapshotCollector?

Reply:
Yes, yes it should.